cloud-functions/cf-dsm-file-scanner/src/main/java/org/broadinstitute/ddp/cf/DSMSomaticFileScanner.java

```diff
@@ -0,0 +1,334 @@
+package org.broadinstitute.ddp.cf;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.InetSocketAddress;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.nio.channels.Channels;
+import java.nio.charset.StandardCharsets;
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.concurrent.ExecutionException;
+
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.cloud.functions.BackgroundFunction;
+import com.google.cloud.functions.Context;
+import com.google.cloud.pubsub.v1.Publisher;
+import com.google.cloud.storage.Blob;
+import com.google.cloud.storage.BlobId;
+import com.google.cloud.storage.Storage;
+import com.google.cloud.storage.StorageOptions;
+import com.google.protobuf.ByteString;
+import com.google.pubsub.v1.ProjectTopicName;
+import com.google.pubsub.v1.PubsubMessage;
+import lombok.AllArgsConstructor;
+import lombok.Value;
+import lombok.extern.flogger.Flogger;
```

Comment:
I realize this is a cut and paste, but I don't think we need Flogger. I don't think it is a problem per se, but we don't use it elsewhere, and it does not seem to be buying us anything.

Reply:
@denniscunningham I agree with you! I tried with Slf4j which is what we use but the logs do not get published correctly. I think it's because Slf4j does not have the same severity levels